arch/xtensa/src/esp32/Kconfig

```
@@ -211,6 +211,40 @@ config ESP32_XTAL_26MHz
 
 endchoice # On-board Crystal Frequency
 
+choice ESP32_RTC_CLK_SRC
+    prompt "RTC clock source"
+    default ESP32_RTC_CLK_SRC_INT_RC
+    help
```

```suggestion
    ---help---
```
nit: minor styling issue

---

done